packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,132 @@
+.rt-RadioCardGroupRoot {
+  user-select: none;
+  cursor: default;
+}
+
+.rt-RadioCardGroupItem {
+  display: flex;
+  align-items: center;
+  cursor: var(--cursor-radio);
```

Comment:
Do we want a separate cursor var for this component?

Reply:
Nope, I'd definitely a button and not radio cursor, that would have to be the same when you have a Card button